pptx/oxml/dml/color.py

```diff
@@ -100,6 +100,9 @@ class CT_ScRgbColor(_BaseColorElement):
     """
     Custom element class for <a:scrgbClr> element.
     """
+    r = RequiredAttribute("r", ST_Percentage)
+    g = RequiredAttribute("g", ST_Percentage)
+    b = RequiredAttribute("b", ST_Percentage)
```

Comment:
Yeah, so SCRGB is just a different way of defining a color, instead of a 0-256 value for RGB, it is a percentage.  I only implemented it as it was used in the MCS slides.